### DISCUSSION TITLE
Crash in Projectile.Shoot

### DISCUSSION THREAD

**Discussion by LK24108** [FakeFishGames/Barotrauma#14756] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

game just crashing random, i dont know for what and i need help.
[crashreport (7).log](https://github.com/user-attachments/files/17202604/crashreport.7.log)

### Reproduction steps

game, in random time just crashing. its want no hapening for hours, or very often.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! The stack trace seems to imply that the game attempted to launch a projectile that has no physics body. This should not be possible in the vanilla game, so I suspect this has something to do with the mods you have enabled.

**Reply by LK24108** [reply]
yes i too thinking about that but what is it?
maybe one of monster shooting projectile which is not be in the game files.
in the last day i dont saw any crash report maybe if i get more i'll leave it here.
if i have mistakes in english sorry.
